ethereum/core/src/test-support/java/org/hyperledger/besu/ethereum/core/BlockDataGenerator.java

```diff
@@ -627,7 +637,9 @@ public static class BlockOptions {
     private Optional<Long> timestamp = Optional.empty();
     private boolean hasOmmers = true;
     private boolean hasTransactions = true;
-    private TransactionType[] transactionTypes = TransactionType.values();
+    private TransactionType[] transactionTypes = {
```

Comment:
Fun fact, you can actually break a few tests just by changing the enum ordering here.